contracts/test/DSTest/test.sol

```diff
@@ -0,0 +1,469 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
was this actually needed? [This example repo](https://github.com/gakonst/foundry-rust-template/blob/master/contracts/src/test/Contract.t.sol) makes it look like you can just import this file using `import "ds-test/test.sol";`

The foundry book says it exists at a [different import](https://book.getfoundry.sh/forge/writing-tests.html?highlight=dstest#writing-tests) though

Reply:
It is part of the foundry standard library but that's not landed yet.  I will plan to update the import once I get that merged in.